pkg/ovalutil/rpm.go

```diff
@@ -24,7 +25,6 @@ func RPMDefsToVulns(ctx context.Context, root oval.Root, protoVuln ProtoVulnFunc
 		Logger()
 	ctx = log.WithContext(ctx)
 	vulns := make([]*claircore.Vulnerability, 0, 10000)
-	pkgcache := map[string]*claircore.Package{}
```

Comment:
why the removal of this?

Reply:
The package cache used a package name as a cache key which doesn't work because the module name is also part of Package struct so I decided to remove it. What do you think is a better solution? And when exactly and how often is this cache used?

Reply:
@Allda can the key be updated to "name" + "module" and the uniqueness of a package retained? 

The cache is just a automatic var, it comes and goes with the invocation of this function, however it keeps us from allocating a ton of pointers.

Reply:
Fixed